core/server/common/src/main/java/alluxio/rocks/RocksStore.java

```diff
@@ -158,10 +163,15 @@ private void formatDbDirs() {
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
-    new File(mDbPath).mkdirs();
   }
 
+  /**
+   * Opens a RocksDB instance or creates a RocksDB if it does not exist.
+   *
+   * @throws RocksDBException
+   */
   private void createDb() throws RocksDBException {
+    new File(mDbPath).mkdirs();
```

Comment:
if the directory is failed to be created, what may happen?

Reply:
If the directory exists, it will do nothing, but it returns false;
If the directory does not exist, a new dir is created, it returns true;
If the directory does not exist, a new dir can not be created, it returns false.

From the return value, we cannot say if the target dir is ready or not.
We need to do check again.
But the following RocksDB.open() will fail if the target dir is not ready.
So, error will be handled there.

Reply:
Explained.